.github/merge_rules.json

```
@@ -1,6 +1,16 @@
 [
    {
-      "name": "ONNX exporter",
+      "name": "Core Maintainers",
+      "patterns": ["*"],
+      "approved_by": ["soumith", "gchanan", "ezyang", "dzhulgakov"],
+      "mandatory_checks_name": [
+         "Facebook CLA Check",
+         "Lint",
+         "pull"
+      ]
+   },
+   {
+      "name": "PyTorch -> ONNX",
```

Please keep ONNX exporter as name
```suggestion
      "name": "ONNX exporter",
```